src/js/views/globals/search/patient-search_views.js

```diff
@@ -0,0 +1,86 @@
+import hbs from 'handlebars-inline-precompile';
+import moment from 'moment';
+import { View } from 'marionette';
+
+import intl from 'js/i18n';
+
+import Picklist from 'js/components/picklist';
+
+import './patient-search.scss';
+
+const EmptyResultsViews = View.extend({
+  tagName: 'li',
+  className: 'patient-search--no-results',
+  template: hbs`{{ @intl.globals.search.patientSearchViews.picklistEmptyView.noResults }}`,
+});
+
+const TipView = View.extend({
+  tagName: 'li',
+  className: 'patient-search--no-results',
+  template: hbs`{{ @intl.globals.search.patientSearchViews.picklistEmptyView.searchTip }}`,
+});
+
+const PatientSearchPicklist = Picklist.extend({
+  className: 'picklist patient-search__picklist',
+  isSelectlist: true,
+  placeholderText: intl.globals.search.patientSearchViews.patientSearchPicklist.placeholderText,
+  getItemFormat(item) {
+    return `${ item.get('first_name') } ${ item.get('last_name') }`;
+  },
+  itemTemplate: hbs`
+    <a class="patient-search__picklist-item{{#if isSelected}} is-selected{{/if}}">
+      {{matchText text query}}{{~ remove_whitespace ~}}
+      <span class="picklist-item__birthdate">{{ birthdate }}</span>
+    </a>
+  `,
+  itemTemplateContext() {
+    return {
+      birthdate: moment(this.model.get('birth_date')).format('MM/DD/YYYY'),
+    };
+  },
+  emptyView() {
```

Comment:
I think you can wrap this in `viewOptions` so we don't need to the picklist component.  Also we could probably add a `template` to the `viewOptions` to avoid needing to override the styles of `input-primary--small`